src/types.ts

```diff
@@ -201,3 +203,19 @@ export interface GenericTemplateAnswer {
   questionId: string;
   questionaryId: number;
 }
+
+export interface Role {
+  shortCode: Roles;
+  title: string;
+  id: number;
+}
+
+export enum Roles {
```

Comment:
Does this mean that we need to add a new role any time a new role is added to the user office? If so, I believe @martin-trajanovski has a PR with a new role for internal reviewers. Once that is merged, then this needs to be updated. Adding this comment just to keep track.

Maybe the role could be just a string to simplify the maintenance. wdyt?

Reply:
@enthusiastio Just wanted to be more specific. But String could be a good idea in this case, since we are not doing any specific operation based on roles in this codebase.